src/Camera.js

```diff
@@ -191,10 +191,18 @@ export default class Camera extends Component {
         message: this.props.permissionDialogMessage,
       });
 
+      // On devices before SDK version 23, the permissions are automatically granted if they appear in the manifest, 
+      // so check and request should always be true.
+      // https://github.com/facebook/react-native-website/blob/master/docs/permissionsandroid.md
+      const isAuthorized = Platform.Version >= 23
+		      ? granted === PermissionsAndroid.RESULTS.GRANTED
+          : granted === true;
```

Comment:
can we do something simpler, like this:

`granted === PermissionsAndroid.RESULTS.GRANTED || granted === true` ?

Reply:
with `Platform.Version >= 23` it looks more strict for me.